plugins/toast/index.ts

```
@@ -1,56 +1,55 @@
     provide: {
-      toast: (props: Props) => {
+      toast: (props: Toast) => {
         const toastNode = document.createElement('div')
-        const vm = createVNode(Dom, { ...props })
+
+        // 限制最大实例数
+        if (instances.length >= MAX_INSTANCES) {
+          const firstId = instances[0].id
+          const oldNode = document.querySelector(`[data-toast-id="${firstId}"]`)
+          if (oldNode && oldNode.parentNode) {
+            oldNode.parentNode.removeChild(oldNode)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)